modules/EnsEMBL/Web/Component/Gene/ComparaOrthologs.pm

```diff
@@ -366,29 +364,26 @@ sub content {
     );
   }   
 
-  if (($hub->action =~ /^Strain_/ && keys %$strains_not_shown)
-    || ($hub->action !~ /^Strain_/ && keys %$species_not_shown)) {
-    my ($total, $no_ortho_species, $strain_refs_html);
-    if ($hub->action =~ /^Strain_/) {
-      $total = scalar keys %$strains_not_shown; 
-      $no_ortho_species = $self->get_no_ortho_species_html($strains_not_shown, $sets_by_species);
-      $strain_refs_html = '';
-    }
-    else {
-      $total = scalar keys %$species_not_shown; 
-      unless ($is_pan) {
-        $no_ortho_species = $self->get_no_ortho_species_html($species_not_shown, $sets_by_species);
-        $strain_refs_html = $self->get_strain_refs_html($strain_refs, $species_not_shown);
-      }
+  if (keys %$species_not_shown) {
+    my $no_ortho_species;
+    my $total = scalar keys %$species_not_shown;
+    unless ($is_pan) {
+      $no_ortho_species = $self->get_no_ortho_species_html($species_not_shown, $sets_by_species);
     }
     my $not_shown_list = $is_pan ? '' : sprintf('<ul id="no_ortho_species">%s</ul>', $no_ortho_species);
+    my $strain_type_breakdown = $self->get_strain_type_breakdown($unshown_strain_types, $total);
+
+    my $not_shown_desc = $total > 1
+                       ? "are not shown in the table above because they don't have any orthologue with"
+                       : "is not shown in the table above because it doesn't have any orthologue with"
+                       ;
+
     $html .= '<br /><a name="list_no_ortho"/>' . $self->_info(
       'Species without orthologues',
```

Comment:
What this looks like if 1 breed lacks orthologues:
![ENSSSCG00110020516_no_ortho_sandbox](https://github.com/user-attachments/assets/d7291317-c265-448b-bfb7-41453b8067f5)

---

What this looks like if many breeds, isolates and species lack orthologues:
![ENSSSCG00000048809_no_ortho_sandbox](https://github.com/user-attachments/assets/cd7a8f7e-83c0-4247-8255-11653d1a0779)